lens/lily/impl.go

```diff
@@ -142,6 +153,59 @@ func (m *LilyNodeAPI) LilyWalk(_ context.Context, cfg *LilyWalkConfig) (schedule
 	return id, nil
 }
 
+func (m *LilyNodeAPI) LilyGapFind(_ context.Context, cfg *LilyGapFindConfig) (schedule.JobID, error) {
+	// the context's passed to these methods live for the duration of the clients request, so make a new one.
+	ctx := context.Background()
+
+	md := storage.Metadata{
+		JobName: cfg.Name,
+	}
+
+	// create a database connection for this watch, ensure its pingable, and run migrations if needed/configured to.
+	db, err := m.StorageCatalog.ConnectAsDatabase(ctx, cfg.Storage, md)
+	if err != nil {
+		return schedule.InvalidJobID, err
+	}
+
+	// TODO add params like walk and watch
```

Comment:
Let's do this TODO. from/to and storage seem the best params to add

Reply:
done.